### ISSUE TITLE
Docker images ?

### ISSUE THREAD

**Issue by kgnfth** [xxcodianxx/youtube-dl-web#39]
Can u create ready to use docker images pls ?

**Comment by greezlu** [comment]
Yeah, I would also be grateful. Like [Gh Packages](https://github.com/features/packages), for example.
